source/sink-connector/configuration-properties/write-strategies.txt

```diff
@@ -48,41 +78,45 @@ Strategies
          default, and the ``InsertOneDefaultStrategy`` if you set the
          ``timeseries.timefield`` option.
        | 
-       | This is the default value for the ``writemodel.strategy`` configuration setting.
+       | This is the default value value for the ``writemodel.strategy``
+         configuration property.
 
    * - | **InsertOneDefaultStrategy**
        | 
      - | **Description:**
        | Insert each sink record into MongoDB as a document. 
-       | Apply the following configuration to your sink connector to specify this setting: 
+       | To specify this strategy, set the configuration property to the
+         following class name:
        
        .. code-block:: properties
            
-          writemodel.strategy=com.mongodb.kafka.connect.sink.writemodel.strategy.InsertOneDefaultStrategy
+          com.mongodb.kafka.connect.sink.writemodel.strategy.InsertOneDefaultStrategy
```

Comment:
Suggestion:
I think the red text (from the syntax highlighting) can be distracting and wonder if "none" might be a better option than "properties".